health/service.go

```diff
@@ -0,0 +1,151 @@
+package health
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+)
+
+// CheckPath is the baked in SOH endpoint path.
+const CheckPath = "/soh"
```

Comment:
Is this exported on purpose?
Is ECS always going to be looking for `/soh` ?

Reply:
because our existing applications use this, it can specified as parameter in the check function